### ISSUE TITLE
Bugs in Demo Programs of `Sparse Convolution`

### ISSUE THREAD

**Issue by zhangzw12319** [traveller59/spconv#539]
https://github.com/traveller59/spconv/blob/master/docs/USAGE.md

In the `USAGE.md` file,

1.
```Python
spconv.SparseConv3d(32, 64, 3), # just like nn.Conv3d but don't support group
nn.BatchNorm1d(64), # non-spatial layers can be used directly in SparseSequential.
nn.ReLU(),
```
it should add paddings for kenel=3, otherwise the spatial shape cannot keeps the same. For example, chenge it into
`spconv.SparseConv3d(32, 64, 3, padding=1)`
or
`spconv.SparseConv3d(32, 64,(3,1,3), padding=(1,0,1)) ` (e.g. Cylinder3D Assymetic Convolution, each dimiension is padded 1 when corresponded kernel is 3) 

2.
```Python
spconv.ToDense(), # convert spconv tensor to dense and convert it to NCHW format.
nn.Conv3d(64, 64, 3),
nn.BatchNorm1d(64),
nn.ReLU(),
```
For me, nn.BatchNorm1d(64) is not applicable, since after `ToDense()` the shape of tensor becomes `(BatchSize, 64, H, W, C)` for example and it is a 5d tensor. Pytorch will report "ValueError('expected 2D or 3D input (got 5D input)".

3. The whole demo progam may cause OOM, since  ` spconv.SparseConvTranspose3d(64, 64, 3, 2)` occupies much GPU memory. For example, when I create 1000 points with bs=1, it eats up all of my 24G memory for RTX 3090. When I comment this line, the memort consumption is 4.3G.
